keyboards/montex/keymaps/default/config.h

```diff
@@ -0,0 +1,62 @@
+/* Copyright 2021 NachoxMacho   
+*   
+* This program is free software: you can redistribute it and/or modify   
+* it under the terms of the GNU General Public License as published by   
+* the Free Software Foundation, either version 2 of the License, or   
+* (at your option) any later version.   
+*   
+* This program is distributed in the hope that it will be useful,   
+* but WITHOUT ANY WARRANTY; without even the implied warranty of   
+* MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the   
+* GNU General Public License for more details.   
+*   
+* You should have received a copy of the GNU General Public License   
+* along with this program.  If not, see &lt;http://www.gnu.org/licenses/&gt;.   
+*/
+
+#pragma once
+
+#include "config_common.h"
+
+/* USB Device descriptor parameter */
+#define VENDOR_ID       0xFEED
+#define PRODUCT_ID      0x6060
+#define DEVICE_VER      0x0001
```

Comment:
```suggestion
#define VENDOR_ID    0xFEED
#define PRODUCT_ID   0x6060
#define DEVICE_VER   0x0001
#define MANUFACTURER %YOUR_NAME%
#define PRODUCT      %KEYBOARD%
```
Please fill these out.

Reply:
Done and added